test/Microsoft.ML.CodeGenerator.Tests/UtilTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.CodeGenerator.Utilities;
+using Microsoft.ML.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.CodeGenerator.Tests
+{
+    public class UtilTest : BaseTestClass
+    {
+        public UtilTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void TestGenerateSampleData()
+        {
+            var data = new[]
+            {
+                new
+                {
+                    Label = "label1",
+                    STR = "feature1",
+                    INT = 2,
+                    DOUBLE = 1.2,
+                    FLOAT = 1.223E+10F,
+                    TrickySTR = "ab\"\';@#$%^&-++==",
```

Comment:
Some others to test:
* `Single.NaN`
* `Single.PositiveInfinity`
* "" (empty string)
* 1 (as boolean)
* "T" (as boolean)

Reply:
### Resolved

Reply:
Thanks!